<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Aeropuerto</class>
 <widget class="QMainWindow" name="Aeropuerto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aeropuerto</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>-&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>&lt;-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>631</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Clase</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Hora</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Vuelo</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="tf_clase"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Compra de Boletos</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Numero de Asiento</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="tf_numAsiento"/>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Numero de vuelo</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="lb_destino">
          <property name="text">
           <string>NULL</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="lb_hora">
          <property name="text">
           <string>NULL</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Avion</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="lb_avion">
          <property name="text">
           <string>NULL</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Destino</string>
          </property>
         </widget>
        </item>
        <item row="9" column="3">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Comprar</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="lb_numVuelo">
          <property name="text">
           <string>NULL</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Salida</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLabel" name="lb_salida">
          <property name="text">
           <string>NULL</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuCrear_Vuelo">
     <property name="title">
      <string>Manejo Vuelo</string>
     </property>
     <addaction name="actionAgregar_Vuelo"/>
     <addaction name="actionModificar_Vuelo"/>
     <addaction name="actionEliminar_Vuelo"/>
    </widget>
    <widget class="QMenu" name="menuCrear_Avion">
     <property name="title">
      <string>Manejo Avion</string>
     </property>
     <addaction name="actionAgregar_Avion"/>
     <addaction name="actionModificar_Avion"/>
     <addaction name="actionEliminar_Avion"/>
    </widget>
    <addaction name="menuCrear_Vuelo"/>
    <addaction name="menuCrear_Avion"/>
    <addaction name="actionCalcular_Horas"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionAgregar_Vuelo">
   <property name="text">
    <string>Agregar Vuelo</string>
   </property>
  </action>
  <action name="actionModificar_Vuelo">
   <property name="text">
    <string>Modificar Vuelo</string>
   </property>
  </action>
  <action name="actionAgregar_Avion">
   <property name="text">
    <string>Agregar Avion</string>
   </property>
  </action>
  <action name="actionCalcular_Horas">
   <property name="text">
    <string>Calcular Horas</string>
   </property>
  </action>
  <action name="actionEliminar_Vuelo">
   <property name="text">
    <string>Eliminar Vuelo</string>
   </property>
  </action>
  <action name="actionModificar_Avion">
   <property name="text">
    <string>Modificar Avion</string>
   </property>
  </action>
  <action name="actionEliminar_Avion">
   <property name="text">
    <string>Eliminar Avion</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
